hydra-node/src/Hydra/Network/Etcd.hs

```diff
@@ -182,8 +186,9 @@ withEtcdNetwork tracer protocolVersion config callback action = do
   -- XXX: Could use TLS to secure peer connections
   -- XXX: Could use discovery to simplify configuration
   -- NOTE: Configured using guides: https://etcd.io/docs/v3.5/op-guide
-  etcdCmd =
-    setCreateGroup True -- Prevents interrupt of main process when we send SIGINT to etcd
+  etcdCmd envVars =
+    setEnv (Map.toList $ onlyEtcdVars envVars <> defaultEnv)
```

Comment:
Must: mappend the `envVars` on top of the `defaultEnv`

Otherwise an `ETCD_AUTO_COMPACTION_MODE` from `envVars` would be overwritten.